ui/src/virtualconsole/vcmatrix.cpp

```diff
@@ -746,7 +746,7 @@ void VCMatrix::slotUpdate()
 
     // Custom Buttons
     for (QHash<QWidget *, VCMatrixControl *>::iterator it = m_controls.begin();
-            it != m_controls.end(); ++it)
+         it != m_controls.end(); ++it)
```

Comment:
i guess you want to rebase this branch on top of master first. there are 5 conflicts including this line

Reply:
GitHub says the PR can be merged...I guess it will automatically rebase and resolve?

Reply:
Could be so, yes. Honestly, I never used git's merge feature. At my work we always rebase and merge without merge commits to keep the history clean and straight. when you do git rebase -i origin/master, you also have a chance to squash related commits, and remove temporary ones